src/main/java/se/liu/ida/hefquin/cli/modules/ModFederation.java

```diff
@@ -287,4 +290,14 @@ protected URI verifyExpectedURI( final String uriString ) {
 		return uri;
 	}
 
+	protected boolean verifyValidVocabMappingFile(final String pathString ) {
+		File f = new File(pathString);
+		if ( f.exists() ){
+			// TODO: .nt file?
```

Comment:
No need to verify that it is an NTriples file. The Jena code that will later be used to parse this file should be able to figure out which file format it is and, then, use the corresponding parser.